src/LibraryUtilities.ts

```diff
@@ -640,3 +640,37 @@ export function getHighlightedText(text: string, highlightedText: string, matchD
 
     return spans;
 }
+
+/**
+ * Find an item from all Items based on path to the item.
+ * 
+ * @param {ItemData[]} pathToItem
+ * An arry of ItemData which represents the path to an item.
+ * 
+ * For example, in the following tree, pathToItem for D would be [A, B, D]
+ * - A
+ *   |- B
+ *      |- C
+ *      |- D
+ * 
+ * @param {ItemData} allItems
+ * An array of ItemData, which contains all the items. In the example above,
+ * allItems would be [A]
+ * 
+ * @return {boolean} true if an item is found, false otherwise
+ */
+export function findItemByPath(pathToItem: ItemData[], allItems: ItemData[]): boolean {
```

Comment:
Good documentation, it is from this we can better understand what the function does. The function has a side-effect (item expanded if found), but the name does not suggest that. So I think the name should be `findAndExpandItemByPath`.